<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LTIN272087" ignored="0" name="Regression Tests" tests="0" failures="0" timestamp="23 Apr 2021 13:43:58 GMT" time="4.229" errors="0">
  <testcase name="@AfterClass afterClass" time="0.058" classname="com.cucumbercraft.runners.RunCucumberTests_Regression">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.cucumber.listener.Reporter.loadXMLConfig(Reporter.java:66)
at com.cucumbercraft.runners.RunCucumberTests_Regression.afterClass(RunCucumberTests_Regression.java:76)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="@AfterClass afterClass" time="0.058" classname="com.cucumbercraft.runners.RunCucumberTests_Smoke">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.cucumber.listener.Reporter.loadXMLConfig(Reporter.java:66)
at com.cucumbercraft.runners.RunCucumberTests_Smoke.afterClass(RunCucumberTests_Smoke.java:83)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass afterClass -->
</testsuite> <!-- Regression Tests -->
